grunt/index.d.ts

```diff
@@ -796,8 +796,8 @@ declare namespace grunt {
              *
              * @note taskFunction.apply(scope: grunt.task.IMultiTask<any>, args: any[])
              */
-            registerMultiTask(taskName: string, taskFunction: Function): void
-            registerMultiTask(taskName: string, taskDescription: string, taskFunction: Function): void
+            registerMultiTask(taskName: string, taskFunction: (this: IMultiTask<any>, ...args: any[]) => void): void;
+            registerMultiTask(taskName: string, taskDescription: string, taskFunction: (this: IMultiTask<any>, ...args: any[]) => void): void;
```

Comment:
Ah, I see. Does `T extends any` provide any IDE support? It seemed weird from a code perspective, but if it helps with auto-complete and stuff, I would be okay with the original approach, otherwise, this seems like the closest we can get without breaking people's stuff.